connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorMaker.java

```diff
@@ -332,11 +303,6 @@ public void run() {
         }
     }
 
-    public ConnectorStateInfo connectorStatus(SourceAndTarget sourceAndTarget, String connector) {
-        checkHerder(sourceAndTarget);
-        return herders.get(sourceAndTarget).connectorStatus(connector);
-    }
```

Comment:
It's a pretty big change in the internal API to make the herders map public. I think we should stick to methods like this one that restrict access to herders. We could retain this method and add a new `taskConfigs` method, which I think should cover all current needs.